JitsiMeetJS.ts

```diff
@@ -106,6 +110,16 @@ interface ICreateLocalTrackFromMediaStreamOptions {
     videoType?: VideoType;
 }
 
+export interface IJoinConferenceOptions {
+    conferenceOptions?: any;
+    connectionOptions?: any;
+    jaas?: {
+        release?: boolean;
+        useStaging?: boolean;
+    };
+    tracks?: JitsiLocalTrack[];
```

Comment:
Is the tracks part wired up yet? I don't see JitsiConference adding the tracks from` this.options.config.tracks `automatically to the media sessions being created. Is that for later?